client/src/components/layout/ReviewList.js

```diff
@@ -1,25 +1,23 @@
-import React from "react"
-import MovieShow from "./MovieShow"
+import React from "react";
+import MovieShow from "./MovieShow";
 
 const ReviewList = (props) => {
-    const allReviewsArray = props.movieReviews.map(reviews =>{
-        let message = ""
-        if(reviews.spoilerWarning === true){
-            message = "Spoiler Alert!"
-        }
-        return (
-            <div key={reviews.id} className="callout">
-                <h2>{reviews.content}</h2>
-                <p>{reviews.rating}</p>
-                <p> {message}</p>
-            </div>
-        )
-    })
+  const allReviewsArray = props.movieReviews.map((reviews) => {
+    let message = "";
+    if (reviews.spoilerWarning === true) {
+      message = "Spoiler Alert!";
+    }
     return (
-        <>
-            {allReviewsArray}
-        </>
-    )
-}
+      <div className="grid-container">
```

Comment:
this grid-container isn't applying much - it is intended to contain a grid horizontally, however the `grid` is not used beyond this